services/dynamic-scheduler/openapi.json

```diff
@@ -44,6 +44,32 @@
           }
         }
       }
+    },
```

Comment:
you are not increasing API versions with such big changes?

Reply:
minor version change sounds ok